R/r6_class.R

```diff
@@ -539,3 +545,26 @@ R6Class <- encapsulate(function(classname = NULL, public = list(),
 
   generator
 })
+
+wrap_dots_fun <- function(dots_fun, arglist_fun) {
```

Comment:
I think this needs to be encapsulated, as in `wrap_dots_fun <- encapsulate(function(....`.